test/e2e/kubectl.go

```diff
@@ -17,18 +17,13 @@ limitations under the License.
 package e2e
 
 import (
-	"bytes"
 	"encoding/json"
-	"fmt"
-	"os/exec"
+	"errors"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
+	. "github.com/onsi/ginkgo"
```

Comment:
The ginkgo import should be last (with a blank line before) as it was before your change. This is consistent with how it's imported in the rest of the e2e test files.